### PR TITLE
Add Marshal API Marshal.GetExceptionPointers

### PR DESCRIPTION
**Pull request by luqunl**
Fix https://github.com/dotnet/corefx/issues/30677

### CODE REVIEW

File: src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs [hunk 1]
```diff
@@ -538,6 +538,8 @@ public static partial class Marshal
         public static System.Exception GetExceptionForHR(int errorCode) { throw null; }
         public static System.Exception GetExceptionForHR(int errorCode, System.IntPtr errorInfo) { throw null; }
         [System.ComponentModel.EditorBrowsableAttribute((System.ComponentModel.EditorBrowsableState)(1))]
+        public static IntPtr GetExceptionPointers() { throw null; }
```
Comment (stephentoub) [inline src/System.Runtime.InteropServices/ref/System.Runtime.InteropServices.cs]: Is it EditorBrowsableState.Never on desktop as well?

File: src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetExceptionPointersTests.cs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+using Xunit;
+
+namespace System.Runtime.InteropServices.Tests
+{
+    public class GetExceptionPointersTests
+    {
+        [Fact]
+        public void GetExceptionPointers_ReturnsExpected()
+        {
+            Assert.Equal(IntPtr.Zero, Marshal.GetExceptionPointers());
```
Comment (stephentoub) [inline src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetExceptionPointersTests.cs]: Is there any additional validation we can/should do for this?

Comment (luqunl) [reply src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/GetExceptionPointersTests.cs]: I am going to add few additional validation later.

**Comment by danmosemsft** [comment]
@luqunl the build is breaking because UWP does not have this API. Please add it to  src\ApiCompatBaseline.uapaot.txt

**Pull request merged**